### PR TITLE
Code cleanups

### PR DESCRIPTION
**Pull request by jared-improbable**

### CODE REVIEW

File: workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public EntityGameObjectCreator(World world, Dictionary<string, GameObject> initi
                 cachedPrefabs[prefabPath] = prefab;
             }
 
-            var gameObject = GameObject.Instantiate(prefab, position, rotation);
+            var gameObject = Object.Instantiate(prefab, position, rotation);
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs]: What's the reason for this change?

File: ReSharper2017.DotSettings [hunk 2]
```diff
@@ -1,4 +1,6 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+
+
```
[the remark this answers is not in the document]
Comment (jared-improbable) [reply ReSharper2017.DotSettings]: Resharper does this weird empty lines thing, I just sort of let it do its thing.

File: ReSharper2017.DotSettings [hunk 3]
```diff
@@ -86,4 +89,12 @@
...
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateBlankLinesAroundFieldToBlankLinesAroundProperty/@EntryIndexedValue">True</s:Boolean>
-	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateThisQualifierSettings/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
\ No newline at end of file
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateThisQualifierSettings/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Housekeeping/ExcludedProjects/ProjectMasksToIgnore/=Improbable_002EGdk_002EGenerated_002Ecsproj/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Housekeeping/ExcludedProjects/ProjectMasksToIgnore/=Improbable_002EGenerated_002ENetworkTypes_002Ecsproj/@EntryIndexedValue">True</s:Boolean>
```
[the remark this answers is not in the document]
Comment (jared-improbable) [reply ReSharper2017.DotSettings]: Nope

File: workers/unity/Assets/Playground/Scripts/Bootstrap.cs [hunk 4]
```diff
@@ -37,26 +36,18 @@ public void Awake()
...
 
-                connectionConfig = new ReceptionistConfig();
-                connectionConfig.UseExternalIp = workerConfigurations.UseExternalIp;
+                connectionConfig = new ReceptionistConfig { UseExternalIp = workerConfigurations.UseExternalIp };
 #endif
             }
             else
             {
-                var commandLineArguments = System.Environment.GetCommandLineArgs();
-                Debug.LogFormat("Command line {0}", string.Join(" ", commandLineArguments.ToArray()));
```
[the remark this answers is not in the document]
Comment (jared-improbable) [reply workers/unity/Assets/Playground/Scripts/Bootstrap.cs]: The command line is the first thing that's printed by Unity in the log...

[the remark this answers is not in the document]
Comment (jared-improbable) [reply workers/unity/Assets/Playground/Scripts/Bootstrap.cs]:
```LICENSE SYSTEM [2018813 14:23:1] Next license update check is after 2018-08-14T11:04:59


LICENSE SYSTEM [2018813 14:23:1] 00331-20090-00000-AA801 != 00331-20090-00000-AA019

Built from '2018.2/staging' branch; Version is '2018.2.0f2 (787658998520) revision 7894616'; Using compiler version '191225831'
OS: 'Windows 10  (10.0.0) 64bit' Language: 'en' Physical Memory: 32706 MB
BatchMode: 1, IsHumanControllingUs: 0, StartBugReporterOnCrash: 0, Is64bit: 1, IsPro: 1
[Package Manager] Server::Start -- Port 49305 was selected

 COMMAND LINE ARGUMENTS:
C:\Program Files\Unity\Hub\Editor\2018.2.0f2\Editor\Unity.exe
-batchmode
-projectPath
E:/git/UnityGDK/workers/unity
-runTests
-testPlatform
editmode
-logfile
E:/git/UnityGDK/logs/unity-editmode-test-run.log
-testResults
E:/git/UnityGDK/logs/editmode-test-results.xml```

File: workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs (hunk 1 above)
Comment (jared-improbable) [reply workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs]: Resharper complains about accessing a super-class' method via a subclass. I can also just suppress the warning

File: workers/unity/Assets/Playground/Scripts/Player/MoveLocalPlayerSystem.cs [hunk 5]
```diff
@@ -24,17 +32,16 @@ public struct NewPlayerData
...
-        [Inject] private NewPlayerData newPlayerData;
-
         public struct PlayerInputData
         {
             public readonly int Length;
             public ComponentArray<Rigidbody> Rigidbody;
             public ComponentDataArray<SpatialOSPlayerInput> PlayerInput;
             public ComponentDataArray<Authoritative<SpatialOSTransform>> TransformAuthority;
-            public ComponentDataArray<Speed> Speed;
+            public ComponentDataArray<Speed> SpeedData;
```
Comment (jared-improbable) [inline workers/unity/Assets/Playground/Scripts/Player/MoveLocalPlayerSystem.cs]: The field `Speed` hides the struct named `Speed` above.

File: workers/unity/Assets/Playground/Scripts/Bootstrap.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (jamiebrynes7) [reply workers/unity/Assets/Playground/Scripts/Bootstrap.cs]: Oh right - had a brain fart about printing these in the Editor 🤦‍♂️

File: workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs (hunk 1 above)
Comment (jamiebrynes7) [reply workers/unity/Assets/Playground/Scripts/EntityInitialization/EntityGameObjectCreator.cs]: Hmm - I'd say `GameObject.Instantiate...` is the more canonical way in Unity-land. But if Resharper complains then its probably fine

File: workers/unity/Assets/Playground/Scripts/Bootstrap.cs [hunk 6]
```diff
@@ -17,7 +17,7 @@ public class Bootstrap : MonoBehaviour
 
         private const int TargetFrameRate = -1; // Turns off VSync
 
-        public const string LoggerName = "Bootstrap";
+        public const string LoggerName = nameof(Bootstrap);
```
Comment (jessicafalk) [inline workers/unity/Assets/Playground/Scripts/Bootstrap.cs]: we should probably define whether we prefer strings or nameof. In another PR of mine, some people preferred to use strings directly, though I would say that nameof is less error-prone

Comment (jared-improbable) [reply workers/unity/Assets/Playground/Scripts/Bootstrap.cs]: Yeah, a string constant isn't refactor-proof, whereas `nameof` is.
